### PR TITLE
Prevent tours from kicking off within iframes

### PR DESCRIPTION
**Pull request by dannon**
... causing duplicate tour step display.  We may want to revisit it later (as commented in onload) because this is potentially useful.  Right now it just causes a display error though.

@guerler This fixes the scratchbook tour issue documented in #1506.

### CODE REVIEW

File: client/galaxy/scripts/onload.js [hunk 1]
```diff
@@ -170,11 +170,27 @@ $(document).ready( function() {
...
+            try {
+                if (window.self !== window.top){
+                    in_iframe = true;
+                } else {
+                    in_iframe = false;
+                }
+            } catch (e) {
+                in_iframe = false;
+            }
+            if (!in_iframe){
```
Comment (guerler) [inline client/galaxy/scripts/onload.js:183]: Is this the same as `if (!window || window.self === window.top)`?

Comment (dannon) [inline client/galaxy/scripts/onload.js:183]: You tell me.  I saw the style I used elsewhere, wasn't sure if the exception handling was needed for anything other than window existence.

Comment (guerler) [inline client/galaxy/scripts/onload.js:183]: I think it is.

Comment (guerler) [inline client/galaxy/scripts/onload.js:183]: Since this is not essential. I leave it up to you :+1: and thanks for the fix.

Comment (martenson) [inline client/galaxy/scripts/onload.js:183]: Replacing 11 lines of `var,try,if,else,catch,if` with a single `if,||` seems like a very good deal to me.

Also I think you can do `if(self==top)` to check that you are not in an (i)frame

Comment (dannon) [inline client/galaxy/scripts/onload.js:183]: Not super thrilled about the level of nitpicking here.  I do find it pretty amusing that guerler said "I think it is" [the same], and you said "I think you can do...", where the code I wrote is fairly explicit and self-documenting.  That was kinda the point.  I'll go open another PR to change this not-important-at-all difference, though.

**Pull request merged**